src/components/Form/Form.tsx

```diff
@@ -58,7 +57,13 @@ const Form = <T extends FormDetails>({
     event.stopPropagation();
 
     if (validate) {
-      const errors = omitBy(validate(state.form), isEmpty) as FormErrors<T>;
+      const errors = Object.fromEntries(
+        Object.entries(validate(state.form)).filter(
+          ([_key, value]) =>
+            value !== "" && value !== null && value !== undefined,
```

Comment:
`== null` is same as doing `=== null || === undefined`

```suggestion
          ([_, value]) => value !== "" && value != null,
```